sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/SpringAiOrchestrationController.java

```diff
@@ -47,30 +37,22 @@ Flux<String> streamChatCompletion() {
   }
 
   @GetMapping("/template")
-  ResponseEntity<String> template(
-      @Nullable @RequestHeader(value = "accept", required = false) final String accept)
-      throws JsonProcessingException {
+  Object template(@Nullable @RequestParam(value = "view", required = false) final String view) {
```

Comment:
(MInor/Comment)

Bit of discussion starter... I probably would've preferred `"format"` over `"view"` since "Views" are a reserved keyword for Spring MVC framework, and this could lead to confusion.